detector_test.go

```diff
@@ -42,14 +41,13 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
 
 		buildpacksDir := filepath.Join("testdata", "by-id")
 
-		outLog = &bytes.Buffer{}
 		config = &lifecycle.DetectConfig{
 			FullEnv:       append(os.Environ(), "ENV_TYPE=full"),
 			ClearEnv:      append(os.Environ(), "ENV_TYPE=clear"),
 			AppDir:        appDir,
 			PlatformDir:   platformDir,
 			BuildpacksDir: buildpacksDir,
-			Logger:        mocks.NewMockLogger(io.MultiWriter(outLog, it.Out())),
+			Logger:        &log.Logger{Handler: logHandler},
```

Comment:
Did you intend to drop the test output (`it.Out()`)?

Reply:
We haven't been writing the log output to `it.Out()` in the other tests files. We should probably make them all consistent one way or another.